src/lib/list/list.scss

```diff
@@ -182,3 +195,31 @@ $mat-dense-list-icon-size: 20px;
     }
   }
 }
+
+.mat-selection-list {
+  a {
+    text-decoration: none;
+    color: inherit;
+  }
+
+  .mat-list-item-content {
```

Comment:
I don't think you need these rules for `mat-list-item-content` and `mat-list-item-content-reverse` here